### ISSUE TITLE
[Bug]: Table td classes unable to be customized

### ISSUE THREAD

**Issue by eblount** [tasteui/tasteui#209]
### What Happened

It seems the classes for table td tags are hardcoded:
https://github.com/tallstackui/tallstackui/blob/935399ea4574e1a5347fe7f0b6f5532de83c216e/src/resources/views/components/table/index.blade.php#L63-L67

### How to Reproduce

```php
TallStackUi::personalize()
    ->table()
    ->block('table.td')
    ->remove('text-sm');
```

### Laravel Version

10.39

### Livewire Version

3.3.3

### TallStackUI Version

1.8.1

### PHP Version

8.2.14

### Operation System

Linux

**Comment by devajmeireles** [comment]
It's good to know we have people like you with us. Feel invited to join us on the Discord server: https://discord.gg/sfr66WXh4A

Solved at #210 , just waiting for a new release.

**Comment by devajmeireles** [comment]
Tag 1.8.2 released! Make sure to upgrade your Composer dependencies. Thanks.
